developerGuide.t2t

```diff
@@ -228,6 +229,44 @@ Events will be covered in greater detail later.
 
 As with other examples in this guide, remember to delete the created app module when you are finished testing and then restart NVDA or reload plugins, so that original functionality is restored.
 
+++ App modules for hosted apps ++
+Some executables host various apps inside.
+These include javaw.exe for running various java programs and wwahost.exe for web apps in Windows 8 and later.

```

Comment:
I think we need to be more specific than just "web apps" here. I'm concerned people might think we're referring to web apps even on the web. Perhaps we could say "some web apps in the Windows Store"? Note "some" as well, because not all web apps in the Store are hosted in wwahost.